CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * [ENHANCEMENT] Query-Frontend and Query-Scheduler: split tenant query request queues by query component with `query-frontend.additional-query-queue-dimensions-enabled` and `query-scheduler.additional-query-queue-dimensions-enabled`. #6772
 * [ENHANCEMENT] Store-gateway: include more information about lazy index-header loading in traces. #6922
 * [ENHANCEMENT] Distributor: support disabling metric relabel rules per-tenant via the flag `-distributor.metric-relabeling-enabled` or associated YAML. #6970
+* [ENHANCEMENT] Distributor: `-distributor.remote-timeout` is now accounted from the first ingester push request being sent. #6972
```

Comment:
Can user notice any difference in practice?

Reply:
If the number of series is really high, the preceding preparation code could consume the timeout before we even start sending network requests.

Maybe we could list this as a bugfix? 🤷

Reply:
Thanks for explanation. Bugfix or enhancement both make sense to me.

I wonder if we should have timeout for entire push request (incl. processing, this will especially become important for OTLP requests), although in fact there is one already -- `-server.http-write-timeout`. ("write" is unrelated to "push" in this case).